docs/assets/css/custom/_variables.scss

```diff
@@ -2,3 +2,4 @@ $inso-purple: rgba(64,0,191,var(--tw-bg-opacity));
 $mid-purple: #530ae4;
 $hover-purple: #4c2991;
 $light-gray: rgba(243,244,246,var(--tw-bg-opacity));
+$btn-focus-box-shadow: unset;
```

Comment:
Removes the box-shadow when clicking on a section header

![image](https://user-images.githubusercontent.com/4312346/133528860-feca3cc4-d1bb-4bdf-909f-7eb9da2ae88a.png)

Reply:
Has been reverted